src/main/java/de/retest/recheck/ignore/SearchFilterFiles.java

```diff
@@ -23,4 +42,15 @@ private SearchFilterFiles() {}
 				.collect( Collectors.toList() ); //
 	}
 
+	public static List<File> getProjectFilterFiles() {
+		try ( Stream<Path> paths = Files.walk( resolveFilterPath() ) ) {
+			return paths.filter( Files::isRegularFile ) //
+					.filter( file -> file.toString().endsWith( ".filter" ) ) //
+					.map( Path::toFile ) //
+					.collect( Collectors.toList() ); //
+		} catch ( final Exception e ) {
+			log.info( "You can create your own filter files in the \\.retest\\filter folder" );
+		}
+		return new ArrayList<>();
```

Comment:
Move `return` in the `catch` block because it is only for this case, then the code is better readable.